packages/manager/apps/dedicated/client/app/dedicated/server/index.js

```diff
@@ -0,0 +1,66 @@
+import angular from 'angular';
+
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-ui-angular';
+
+import bandwidth from './bandwidth/bandwidth.module';
+import dashboard from './dashboard';
+import interfaces from './interfaces/interfaces.module';
+import monitoring from './monitoring';
+import servers from './servers/servers.module';
+
+import bandwidthVrackOrderService from './server.bandwidth-vrack-order.service';
+import component from './server.component';
+import featureAvailability from './server.feature-availability';
+import routing from './server.routing';
+import service from './server.service';
+
+import './server.less';
+
+const moduleName = 'ovhManagerDedicatedServer';
+
+angular
+  .module(moduleName, [
+    bandwidth,
+    dashboard,
+    interfaces,
+    monitoring,
+    'oui',
+    'pascalprecht.translate',
+    servers,
+    'ui.router',
+  ])
+  .component('dedicatedServer', component)
+  .config(routing)
+  .service('Server', service)
+  .service('BandwidthVrackOrderService', bandwidthVrackOrderService)
+  .service('DedicatedServerFeatureAvailability', featureAvailability)
+  .constant('FIREWALL_RULE_ACTIONS', {
+    ALLOW: 'PERMIT',
+    DENY: 'DENY',
```

Comment:
I think those should be declared in a constant file and exported. That way we can use them outside of angularjs if we want to

Reply:
Yes but I chose not to do it with this refactoring, I can't do everything :)